client/my-sites/plugins/access-control.js

```diff
@@ -48,4 +39,4 @@ const hasRestrictedAccess = ( site ) => {
 	}
 };
 
-export default { hasRestrictedAccess };
+export default { getHasRightsToManagePlutins, hasRestrictedAccess };
```

Comment:
I know that this existed before, but I wanted to note that this is unusual export usage. We normally do this inline